### PR TITLE
update meta data for release 4.1.0

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -2768,6 +2776,23 @@
     <providesextension>couchbase</providesextension>
     <extsrcrelease/>
     <changelog>
+        <release>
+            <date>2022-05-11</date>
+            <version>
+                <release>4.0.0</release>
```
Comment (davidkelly) [inline package.xml:2782]: should this be 4.1.0?

**Review by davidkelly — CHANGES_REQUESTED** [review]
I'm not sure about the format of the package.xml, but it looked like there was a typo.

File: package.xml (hunk 1 above)
Comment (avsej) [reply package.xml:2782]: no, this is changelog, it includes old release information.

**Review by raycardillo — APPROVED** [review]
LGTM

**Review by davidkelly — APPROVED** [review]

**Pull request merged**